internal/internal/src/footer.tsx

```diff
@@ -20,9 +20,17 @@ const footerProducts = [
     name: "ENSRainbow",
     href: "https://ensrainbow.io",
   },
+  {
+    name: "ENSAdmin",
+    href: "https://admin.ensnode.io",
+  },
+  {
+    name: "ENS Referral Programs",
+    href: "https://namehashlabs.org/ens-v2-referral-programs",
```

Comment:
```suggestion
    href: "https://namehashlabs.org/ens-referral-program",
```

I think we are keeping the route unchanged, right?

Reply:
@lightwalker-eth I changed the route and created a redirect from `/ens-referral-program` to `/ens-v2-referral-programs`